src/cmd/cmd_query.go

```diff
@@ -286,13 +291,11 @@ func RunQueryCmdLine() {
 
 		start := time.Now()
 		// We can load and query at the same time
-		if *flags.LOAD_AND_QUERY {
-			t.LoadAndQueryRecords(flags, &loadSpec, &querySpec)
+		t.LoadAndQueryRecords(flags, &loadSpec, &querySpec)
 
-			end := time.Now()
-			sybil.Debug("LOAD AND QUERY RECORDS TOOK", end.Sub(start))
-			querySpec.PrintResults(flags)
-		}
+		end := time.Now()
+		sybil.Debug("LOAD AND QUERY RECORDS TOOK", end.Sub(start))
+		querySpec.PrintResults(*flags.OP, printConfig)
```

Comment:
why flags.OP not in printConfig?

Reply:
It wasn't used in all the places so I wasn't sure it belonged.

Reply:
"better stuffed than sorry"